app/views/site/_study_settings_general.html.erb

```diff
@@ -0,0 +1,28 @@
+<h3>General Settings</h3>
+<p class="help-block">
+  Edit general study properties, such as public visibility and data release dates.
+</p>
+<div class="form-group row">
+  <div class="col-md-12">
+    <%= f.label :name %><br/>
+    <%= f.text_field :name, class: 'form-control', autocomplete: 'off' %>
+  </div>
+</div>
+<div class="form-group row">
+  <div class="col-md-4">
+    <%= f.label :embargo, 'Data release date' %>&nbsp;
+    <span class='fas fa-question-circle' data-toggle='tooltip'
+          title='Setting a data release date will prevent downloads for everyone except study owners & shared users,
+          and will expire at 12AM on the date specified (leave blank to allow downloads)' data-placement='right'>
```

Comment:
Styling as "AM" or "A.M." or "am" or "a.m." is generally pretty unsettled (I'm glad we both like the first!), but essentially all products separate the digit from the twelve-hour period.

```suggestion
          and will expire at 12 AM on the date specified (leave blank to allow downloads)' data-placement='right'>
```